src/styles/main.scss

```diff
@@ -10,17 +10,78 @@ body {
   font-weight: 900;
 }
 
+.move {
+  &-up {
+    &--1 {
+      transform: translateY(-85px);
+    }
+    &--2 {
+      transform: translateY(-170px);
+    }
+    &--3 {
+      transform: translateY(-255px);
+    }
+  }
+
+  &-down {
+    &--1 {
+      transform: translateY(85px);
+    }
+    &--2 {
+      transform: translateY(170px);
+    }
+    &--3 {
+      transform: translateY(255px);
+    }
+  }
+
+  &-left {
+    &--1 {
+      transform: translateX(-85px);
+    }
+    &--2 {
+      transform: translateX(-170px);
+    }
+    &--3 {
+      transform: translateX(-255px);
+    }
+  }
+
+  &-right {
+    &--1 {
+      transform: translateX(85px);
+    }
+    &--2 {
+      transform: translateX(170px);
+    }
+    &--3 {
+      transform: translateX(255px);
+    }
+  }
```

Comment:
Consider using SCSS loops and variables for repeated transformations to make the code DRYer and more maintainable. You have a pattern where you're applying `transform` with multiples of a base value (85px). You could define a base value and use a loop to generate these classes.